### DISCUSSION TITLE
How can i send message to other Computer (Windows)  thru lan?

### DISCUSSION THREAD

**Discussion by MizukumaMC** [termux/termux-app#3020] [Q&A] [answered]
Im planning to rickroll the whole computers (windows) on same network but the `msg` command is only for Windows 10 Pro! I dont have Windows PC and only have android phone so im finding a solution to use `msg`command. Im asking a help on the community that how can i do `msg` command on termux and rickroll someone (they using windows OS) with same network. Im new on this app and dont have an idea to do it. Is there any github repo for msg command? Thank you guys,i appreciate your help!

**Comment by 2096779623** [comment, marked as answer]
https://unix.stackexchange.com/questions/144474/write-text-messages-between-unix-and-windows

On termux, you need install `netcat-openbsd` first.
